WordPress/WordPressUITests/Screens/BaseScreen.swift

```diff
@@ -53,7 +53,7 @@ class BaseScreen {
             safari.launch()
 
             // Select the URL bar when Safari opens
-            let urlBar = safari.otherElements["URL"]
+            let urlBar = safari.textFields.firstMatch
```

Comment:
Could we switch this to `let urlBar = safari.textFields["URL"]`? That'll be more precise, and avoid any issues with local runs where there might be another text field on the screen that could be selected instead of the URL field.

Reply:
This doesn't work in iOS 13 – it looks like they changed how the view hierarchy works. The test just stalls out on this line because it can't seem to find a match :(

Reply:
Hm, it works on my end — `otherElements["URL"]` doesn't work anymore but using `textFields["URL"]` everywhere instead was working for me.

Reply:
It's possible there's another factor I didn't notice that's making that work, though. If it doesn't work for you, I'm ok with your change here. It shouldn't ever be an issue on CircleCI in any case and it's pretty obvious what's happening if it fails on a local test run.

Reply:
Ah – I might've been doing something wrong – I think I was still using `otherElements`. Updated this to your suggestion and it works locally – we'll wait for the ✅!

Reply:
So weirdly, this breaks on iPad. In order to get the release moving, I'd like to merge this PR with the less-good version, then address this after, if that's ok with you?

Reply:
Sorry for the delay getting back to you about this one — I'll investigate and follow up on this.